lib/trino-parquet/src/main/java/io/trino/parquet/reader/ChunkedInputStream.java

```diff
@@ -41,21 +42,20 @@
 {
     private final Iterator<? extends ChunkReader> chunks;
     private ChunkReader currentChunkReader;
-    private BasicSliceInput current;
+    private BasicSliceInput current = EMPTY_SLICE.getInput();
```

Comment:
add a comment that we need this becasue current == null means the stream is closed